src/ui/actions/eventListeners/eventListenerUtils.ts

```diff
@@ -73,6 +73,11 @@ export const IGNORABLE_PARTIAL_SOURCE_URLS = [
   "__cypress/runner/",
 ];
 
+export const MORE_IGNORABLE_PARTIAL_URLS = IGNORABLE_PARTIAL_SOURCE_URLS.concat(
+  // Ignore _any_ 3rd-party package for now
+  "node_modules"
```

Comment:
we might want to add the `react-redux` internal url in there too

Reply:
we can do that, and then change `filteredPauseFrames[2]` to `filteredPauseFrames[0]`